google-cloud-storage/samples/acceptance/notification_test.rb

```diff
@@ -26,17 +26,24 @@
 
   before :all do
     @bucket = create_bucket_helper random_bucket_name
-    pubsub = Google::Cloud::Pubsub.new
-    @topic = pubsub.create_topic random_topic_name
-    topic.policy do |p|
-      p.add "roles/pubsub.publisher",
-            "serviceAccount:#{storage_client.service_account_email}"
-    end
+    @pubsub = Google::Cloud::PubSub.new
```

Comment:
Does `pubsub` need to be an instance variable? It isn't used in the `after` block.